util/dvsim/SimCfg.py

```diff
@@ -578,21 +581,24 @@ def gen_results_sub(items, results, fail_msgs):
         if self.revision_string:
             results_str += "### " + self.revision_string + "\n"
 
-        # Add path to testplan.
-        if hasattr(self, "testplan_doc_path"):
-            testplan = "https://" + self.doc_server + '/' + self.testplan_doc_path
-        else:
-            testplan = "https://" + self.doc_server + '/' + self.rel_path
-            testplan = testplan.replace("/dv", "/doc/dv_plan/#testplan")
+        # Add path to testplan, only if it has entries (i.e., its not dummy).
+        if self.testplan.has_entries():
```

Comment:
As I mentioned in the definition of `has_entries`, this might be clearer as just
```python
if self.testplan.entries:
    ...
```
and I think this is a bit more Python and less Java :-)